src/classes/RD2_EnablementService_TEST.cls

```diff
@@ -69,17 +71,20 @@ public with sharing class RD2_EnablementService_TEST {
 
     /****
     * @description Verifies a new Opportunity is not created on Recurring Donations insert
-    * when enhanced Recurring Donations are enabled
+    * when enhanced Recurring Donations are enabled and the RD2 Trigger has been disabled.
+    * This verifies that RD2 functionality is isolated from RD1 functionality.
     */
     @isTest
-    private static void shouldNotCreateNewOppOnInsertWhenEnhancementIsEnabled() {
+    private static void shouldNotCreateNewOppOnInsertWhenEnhancementIsEnabledIfRD2TriggerIsDisabled() {
```

Comment:
Since the AC that was valid before is not valid currently due to new requirements, we do not need this test, thus I would suggest we delete it. The test itself has contradiction in the name. Also, this is a test that tests code not what user expects or what requirements are.

Reply:
I agree that original test method is no longer valid.

However, like the previous test, this validates isolation by ensuring that when RD2 is enabled, the RD1 triggers are not running to create the installment opp. It does this by enabling RD2 and disabling the RD2 trigger. If code isolation is working then neither trigger will execute and no Installment Opp will be created. If code isolation was not working, then the RD1 trigger _might_ run and create the installment opp. 

What would you think about a method name like `shouldNotCreateNewOppOnInsertIfRD2TriggerIsDisabledWhenEnhancedRDIsEnabled`?